tests/dom/getdatafromelement.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* bender-tags: dom, browser-only */
+
+'use strict';
+
+import getDataFromElement from '/ckeditor5/utils/dom/getdatafromelement.js';
+
+describe( 'getDataFromElement', () => {
+	[ 'textarea', 'template', 'div' ].forEach( ( elementName ) => {
```

Comment:
`for( elementName of [ 'textarea', 'template', 'div' ] )` - is not it beautiful?